src/index.ts

```diff
@@ -1,16 +1,27 @@
 import path from "path";
 import cors from 'cors';
-import express from 'express';
+import express, { Request } from 'express';
 import { getImage, getDomain } from './domain';
 require('dotenv').config()
 
+interface RequestParams {
+  tokenId?: string;
+}
+
 const app = express();
 
 app.get('/', (req, res) => {
   res.send('Well done mate!');
 })
 
-app.get('/name/:tokenId', async function (req, res) {
```

Comment:
Doesn't Express return a 404 by default for URIs with no handler defined?

Reply:
I actually misunderstood the to-do, got updated by @makoto will move the 404 if token doesn't exist. current check is unnecessary unless we want a custom 404 message.